### PR TITLE
Revamp of EIPs page

### PR DESCRIPTION
**Pull request by Pandapip1**
## Description

Fixes a few grammar issues/typos and improves the factual accuracy of the content.

## Related Issue

None

### CODE REVIEW

**Comment by Pandapip1** [comment]
Bump.

File: src/content/eips/index.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-title: Ethereum Improvement Proposals (EIPs)
-description: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+title: Ethereum Improvement Proposals
+description: The basic information you need to understand EIPs
 lang: en
 sidebar: true
 ---
 
-# Introduction to Ethereum Improvement Proposals (EIPs) {#introduction-to-ethereum-improvement-proposals-eips}
+# Introduction to Ethereum Improvement Proposals {#introduction-to-ethereum-improvement-proposals}
```
Comment (minimalsm) [inline src/content/eips/index.md:8]: We'd want to keep this as per our style guide: https://ethereum.org/en/contributing/style-guide/#acronyms

```suggestion
# Introduction to Ethereum Improvement Proposals (EIPs) {#introduction-to-ethereum-improvement-proposals}
```

File: src/content/eips/index.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
 ---
-title: Ethereum Improvement Proposals (EIPs)
-description: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+title: Ethereum Improvement Proposals
```
Comment (minimalsm) [inline src/content/eips/index.md:2]:
```suggestion
title: Ethereum Improvement Proposals (EIPs)
```

**Review by minimalsm — APPROVED** [review]
Thanks @Pandapip1, sorry for the delay 😎

**Pull request merged**